frontend/src/index.html

```diff
@@ -134,6 +134,12 @@
           </button>
           <span id="stats" class="whitespace-nowrap text-sm"></span>
           <span id="loading" class="text-gray-500 text-sm">Loading…</span>
+          <div id="search-error" class="text-gray-500 text-sm search-error-wrapper hidden">
+            Error occurred:
+            <b id="search-error-message"> ${TempStorage.searchError.message} </b>
```

Comment:
The contents of this tag looks like a template token, but in reality it does not work here like this, right? Therefore this syntax is confusing. I think it's better to just keep the tag empty, that would be honest at least.

Reply:
earlier it was empty tag and i was injecting whole HTML from javascript, then you asked to put HTML part to HTML file only. now i updated code accordingly. so i am confused what should be done.

Intention to write this code:
This "div" is hidden by default and has most of the ready-made text except dynamic error message. this tag also includes a link which is redirecting to google form as requested. so when error comes i am just updating "b" tag content only without touching link and static text in this tag.

Reply:
I believe here is a misunderstanding. Sorry for not being clear enough.

Here is what I mean:
```suggestion
            <b id="search-error-message"></b>
```

I'm not asking you to eliminate the `b` tag altogether. I mean that having `${TempStorage.searchError.message}` inside it does not make any sense to me. Therefore no contradiction with what I requested earlier to avoid JS-constructed HTML.

Reply:
now i got it. i will fix it.

Reply:
that was added mistakenly